pkg/crc/adminhelper/hosts_nowin.go

```diff
@@ -3,10 +3,15 @@
 package adminhelper
 
 import (
+	"os"
+
 	crcos "github.com/code-ready/crc/pkg/os"
 )
 
 func execute(args ...string) error {
+	if _, err := os.Stat(adminHelperPath); os.IsNotExist(err) {
```

Comment:
I don't think the `os.IsNotExist` check is useful here, if we cannot stat the helper, I doubt we'll be able to run it.